easybuild/tools/filetools.py

```diff
@@ -583,11 +591,18 @@ def guess_patch_level(patched_files, parent_dir):
             if os.path.isfile(os.path.join(parent_dir, *tf2[level:])):
                 path_found = True
                 break
+	    else:
+		# If the file is missing, we create an empty file,
+		# but only if the old line is /dev/null
+		if patched_file_old in ['/dev/null']:
+		    open(os.path.join(parent_dir, *tf2[level:]), "a+")
```

Comment:
It's needed for this case here (adding a file in the base dir)

--- /dev/null   2015-07-10 16:14:12.096153473 +0200
+++ configure   2015-10-28 16:58:26.708873345 +0100

without the patch I get:

 Can't determine patch level for patch